### PR TITLE
DDCNLS-1863

### PR DESCRIPTION
**Pull request by fred-jackson**
@nehakannaujia  - few problems so deleted the branch and started again. Rebasing message replaced and units updated.

### CODE REVIEW

File: app/views/calculation/costsAtLegislationStart.scala.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 @(data: Form[CostsAtLegislationStartModel])(implicit request: Request[_], messages: Messages)
 
-@main_template(Messages("calc.costsAtLegislationStart.title")) {
+@main_template(Messages("calc.costsAtLegislationStart.title") {
```
Comment (nehakannaujia) [inline app/views/calculation/costsAtLegislationStart.scala.html]: Is this bracing closed? Seems like one brace is deleted ?

Comment (fred-jackson) [reply app/views/calculation/costsAtLegislationStart.scala.html]: This was where I made the mistake, deleted the branch (which shouldn't be showing but is) and then created a new branch with the proper changes.

Comment (fred-jackson) [reply app/views/calculation/costsAtLegislationStart.scala.html]: I've just re-run the unit tests and they're okay.

**Comment by jameshforster** [comment]
Closed until it can be addressed by a story on DDCTs backlog.

**Pull request closed without merging**